tests/admin.test.ts

```diff
@@ -182,3 +185,34 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('event creation', () => {
```

Comment:
Since you have created a new category of test, lets also add event creation success test here as well. Make sure to reference how other tests approach this "success" kind of tests.